iroh-net-report/src/lib.rs

```diff
@@ -740,7 +863,7 @@ struct ReportRun {
 /// If successful this returns the bound socket and will forward STUN responses to the
 /// provided *actor_addr*.  The *cancel_token* serves to stop the packet forwarding when the
 /// socket is no longer needed.
-fn bind_local_stun_socket(
+pub fn bind_local_stun_socket(
```

Comment:
do we have to make this public?

Reply:
maybe just under a test_utils flag?

Reply:
Yea good point. It's useful for `iroh doctor` but anyone else attempting a net report for production use would likely do something similar to what we currently do in iroh & pass in the specific socket they are using.